### PR TITLE
C_CPP : Arrays in CPP#937

### LINKED ISSUE #937: C_CPP: Arrays in CPP
<hr>

## Description 📜

<hr>

## Domain of Contribution 📊

- [x] C/CPP

<hr>

### PR DESCRIPTION
**Pull request by Aaryan376**
<hr>

## Description 📜
 
Fixes  #937
Resolved #937

Arrays in CPP- added documentation

<hr>

## Type of change 📝

- [x] Documentation (Content Creation in the form of codes or tutorials)

<hr>

## Domain of Contribution 📊

- [x] C/CPP

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Review by deeqakkk — APPROVED** [review]
Informative and easy to understand.💯👏

File: C_CPP/Arrays/readme.md [hunk 1]
```diff
@@ -1 +1,5 @@
 
+
+
+Arrays in CPP 
+https://github.com/Aaryan376/winter-of-contributing/tree/C_CPP/C_CPP/Arrays/ArrayIntroduction
```
Comment (Saviour1001) [inline C_CPP/Arrays/readme.md:5]: Add a relative link here and not an absolute one. 
[Here](https://stackoverflow.com/questions/7653483/github-relative-link-in-markdown-file) is how to do that.

Comment (Aaryan376) [reply C_CPP/Arrays/readme.md]: I have changed it kindly check.

**Review by Suvraneel — CHANGES_REQUESTED** [review]
Observe the changes

Remove C_CPP/Arrays/readme.md

**Comment by Aaryan376** [comment]
> Observe the changes
> 
> Remove C_CPP/Arrays/readme.md

> I have done the required changes. Kindly check and merge.

**Comment by Aaryan376** [comment]
I have done the changes. Kindly check and merge.

**Comment by Suvraneel** [comment]
@Aaryan376  Nice work 🌊 :octocat:

**Pull request merged**